### PR TITLE
BUG: Issue #29128 Series.var not returning the correct result

### LINKED ISSUE #29128: Variance of nullable Int64 ignores ddof argument
Variance calculation `Series.var()` is ignoring the `ddof` argument for nullable integer type `Int64`. Behavior should match that of `float` type.

```python
>>> pd.__version__
'0.25.1'
>>> np.__version__
'1.17.2'
>>> s1 = pd.Series(data=[1,2,3,4,5,6,np.nan,np.nan])
>>> series.var(ddof=0)
3.5
>>> s1 = pd.Series(data=[1,2,3,4,5,6,np.nan,np.nan])
>>> s1.var(ddof=0)
2.9166666666666665
>>> s1.var(ddof=1)
3.5
>>> s2 = pd.Series(data=[1,2,3,4,5,6,np.nan,np.nan], dtype="Int64")
>>> s2.var(ddof=0)
3.5
>>> s2.var(ddof=1)
3.5 
```

### PR DESCRIPTION
**Pull request by HubertKl**
- [ x ] closes #29128
- [ x ] tests added / passed
- [ x ] passes `black pandas`
- [ x ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ x ] whatsnew entry

This PR closes https://github.com/pandas-dev/pandas/issues/29128

### CODE REVIEW

File: doc/source/whatsnew/v0.25.3.rst [hunk 1]
```diff
@@ -16,6 +16,11 @@ Groupby/resample/rolling
 
 - Bug in :meth:`DataFrameGroupBy.quantile` where NA values in the grouping could cause segfaults or incorrect results (:issue:`28882`)
 
+Series
```
Comment (jreback) [inline doc/source/whatsnew/v0.25.3.rst]: move to 1.0

File: doc/source/whatsnew/v0.25.3.rst [hunk 2]
```diff
@@ -16,6 +16,11 @@ Groupby/resample/rolling
...
+^^^^^^
+
+- Bug in :meth:`Series.var` not computing the right value with an integer series with missing values when ddof = 0
```
Comment (jreback) [inline doc/source/whatsnew/v0.25.3.rst]: add the issue number reference

Comment (jreback) [reply doc/source/whatsnew/v0.25.3.rst]: this goes in the numeric section

File: pandas/core/algorithms.py [hunk 3]
```diff
@@ -1155,7 +1155,7 @@ def compute(self, method):
         n = min(n, narr)
 
         kth_val = algos.kth_smallest(arr.copy(), n - 1)
-        ns, = np.nonzero(arr <= kth_val)
+        (ns,) = np.nonzero(arr <= kth_val)
```
Comment (jreback) [inline pandas/core/algorithms.py]: why this change?

File: pandas/core/arrays/integer.py [hunk 4]
```diff
@@ -652,7 +652,7 @@ def _reduce(self, name, skipna=True, **kwargs):
             data[mask] = self._na_value
 
         op = getattr(nanops, "nan" + name)
-        result = op(data, axis=0, skipna=skipna, mask=mask)
+        result = op(data, axis=0, skipna=skipna, mask=mask, **kwargs)
```
Comment (jreback) [inline pandas/core/arrays/integer.py]: which args are meant to be passed in?

File: pandas/core/frame.py [hunk 5]
```diff
@@ -4829,7 +4829,7 @@ def drop_duplicates(self, subset=None, keep="first", inplace=False):
         duplicated = self.duplicated(subset, keep=keep)
 
         if inplace:
-            inds, = (-duplicated)._ndarray_values.nonzero()
+            (inds,) = (-duplicated)._ndarray_values.nonzero()
```
Comment (jreback) [inline pandas/core/frame.py]: don't change unrelated things

**Review by jreback — CHANGES_REQUESTED** [review]
there are a ton of unrelated changes here. can you just change what is releavnt

File: pandas/core/arrays/integer.py (hunk 4 above)
Comment (HubertKl) [reply pandas/core/arrays/integer.py]: This is supposed to let the ddof argument to be passed in here in order to fix the issue.

File: pandas/core/frame.py (hunk 5 above)
Comment (HubertKl) [reply pandas/core/frame.py]: sorry black made those changes and I blindly accepted. I am changing my version of black into 19.3b0 and rerunning it.

File: doc/source/whatsnew/v0.25.3.rst (hunk 1 above)
Comment (HubertKl) [reply doc/source/whatsnew/v0.25.3.rst]: Done

File: doc/source/whatsnew/v0.25.3.rst (hunk 2 above)
Comment (HubertKl) [reply doc/source/whatsnew/v0.25.3.rst]: Done

File: pandas/core/algorithms.py (hunk 3 above)
Comment (HubertKl) [reply pandas/core/algorithms.py]: Sorry It was changed automatically by black pandas, because I didn't run the right version of the latter.

**Comment by HubertKl** [comment]
Sorry for the mess, I ran the wrong version of black. 

I updated my PR accordingly to the change requested.

File: pandas/core/frame.py (hunk 5 above)
Comment (HubertKl) [reply pandas/core/frame.py]: fixed

File: doc/source/whatsnew/v1.0.0.rst [hunk 6]
```diff
@@ -324,7 +324,7 @@ Numeric
 - :class:`DataFrame` flex inequality comparisons methods (:meth:`DataFrame.lt`, :meth:`DataFrame.le`, :meth:`DataFrame.gt`, :meth: `DataFrame.ge`) with object-dtype and ``complex`` entries failing to raise ``TypeError`` like their :class:`Series` counterparts (:issue:`28079`)
 - Bug in :class:`DataFrame` logical operations (`&`, `|`, `^`) not matching :class:`Series` behavior by filling NA values (:issue:`28741`)
 - Bug in :meth:`DataFrame.interpolate` where specifying axis by name references variable before it is assigned (:issue:`29142`)
--
+- Bug in :meth:`Series.var` not computing the right value with an integer series with missing values when ddof = 0 (:issue:`29128`)
```
Comment (jreback) [inline doc/source/whatsnew/v1.0.0.rst]: say nullable integer dtype

its not ``ddof=0``, rather not passing thru the ``ddof`` arg

**Review by jreback — CHANGES_REQUESTED** [review]
looks good, some comments, ping on green.

File: pandas/tests/arrays/test_integer.py [hunk 7]
```diff
@@ -829,6 +829,14 @@ def test_arrow_array(data):
     assert arr.equals(expected)
 
 
+@pytest.mark.parametrize("ddof", [0, 1])
```
Comment (jreback) [inline pandas/tests/arrays/test_integer.py]: can you parameterize this over kurt, skew, sem as well

File: doc/source/whatsnew/v1.0.0.rst (hunk 6 above)
Comment (HubertKl) [reply doc/source/whatsnew/v1.0.0.rst]: Done

File: pandas/tests/arrays/test_integer.py (hunk 7 above)
Comment (HubertKl) [reply pandas/tests/arrays/test_integer.py]: Done

**Review by HubertKl — COMMENTED** [review]
Changes has been done, checks are green. @jreback

**Comment by jbrockmendel** [comment]
@HubertKl can you rebase

**Comment by HubertKl** [comment]
> @HubertKl can you rebase

Done

**Review by WillAyd — APPROVED** [review]
Lgtm

**Comment by WillAyd** [comment]
Thanks @HubertKl
